rmk-config/src/toml_config.rs

```diff
@@ -122,6 +122,15 @@ pub struct LayoutConfig {
     pub cols: u8,
     pub layers: u8,
     pub keymap: Vec<Vec<Vec<String>>>,
+    pub tri_layer: Option<TriLayerConfig>,
```

Comment:
should we have another field for advanced keyboard config? such as tri-layer/HRM/etc, corresponds to KeyboardOptionConfig？

Reply:
Ok, I was also thinking about giving it a better name instead of `KeyboardOptionConfig`, maybe `BehaviorConfig`?

Reply:
I like this name

Reply:
so the section in toml file should be ［behavior］?

Reply:
yes, but should it be inside the layout section or a separate one?

Reply:
I prefer a separate top level `[behavior]`, since there are lots of non-layer behaviors we can customize